tests/airgapped/pipelines/pipelines-runner/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM python:3.11.9-alpine
```

Comment:
Why dont we use Ubuntu for base image?

Reply:
This image is used for testing purposes only, so the point is to have a small size image that enables us to run the pipeline example. We'd like to reduce the image pull time for testing.